### DISCUSSION TITLE
Installation - PyPolo

### DISCUSSION THREAD

**Discussion by giscus** [Weizhe-Chen/PyPolo#9] [Q&A]
# Installation - PyPolo

https://pypolo.readthedocs.io/en/latest/installation/

**Comment by tonitonitonitoni** [comment]
Hi, I've just installed this, and I'd like to implement your methods in my paper.  I'd really like to run your code.  Do you have any examples of how all of these excellent classes and methods you've collected work together? Thank you

**Comment by DoongLi** [comment]
I think the Pypolo project is currently incomplete.
